mu/locale/es/LC_MESSAGES/mu.po

```diff
@@ -23,23 +23,23 @@ msgstr "Registrando en {}"
 
 #: mu/app.py:183
 msgid "Debugger requires a Python script filename to run."
-msgstr "Para ejecutar el depurador el script de Python requiere un nombre de archivo."
+msgstr "Para ejecutar el depurador, el script de Python requiere un nombre de archivo."
 
 #: mu/debugger/client.py:106
 msgid "Connection timed out. Is your machine slow or busy? Free up some of the machine's resources and try again."
-msgstr "Tiempo de conexión agotado. ¿Es su ordenador lento o esta ocupado? Libere algunos de los recursos del ordenador y vuelva a intentarlo."
+msgstr "Tiempo de conexión agotado. ¿Es tu ordenador lento o esta ocupado? Libere algunos de los recursos del ordenador y vuelva a intentarlo."
 
 #: mu/debugger/client.py:116
 msgid ""
 "Could not find localhost.\n"
 "Ensure you have '127.0.0.1 localhost' in your /etc/hosts file."
 msgstr ""
 "No se pudo localizar localhost.\n"
-"Asegúrese de tener '127.0.0.1 localhost' en su archivo /etc/hosts."
+"Asegúrate de tener '127.0.0.1 localhost' en su archivo /etc/hosts."
 
 #: mu/interface/dialogs.py:55
 msgid "Select Mode"
-msgstr "Selección de modo"
+msgstr "Seleccionar de modo"
```

Comment:
This one doesn't sound quite right to me, did you mean "Seleccionar modo"?
This is the title for this window:

<img width="712" alt="image" src="https://user-images.githubusercontent.com/4189262/48591516-1dad3400-e93c-11e8-873e-68aded5ed953.png">